### PR TITLE
add details about per-machine install

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4508

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ To learn more about Teams and Microsoft 365 Apps for enterprise, see [How to exc
...
+    > These examples also use the **ALLUSERS=1** parameter. When you set this parameter, Teams Machine-Wide Installer appears in Programs and Features in Control Panel and in Apps & features in Windows Settings for all users of the computer. All users can then uninstall Teams if they have admin credentials. It's important to understand the difference between **ALLUSERS=1** and **ALLUSER=1**. The **ALLUSERS=1** parameter can be used in non-VDI and VDI environments and the **ALLUSER=1** parameter is used only in VDI environments to specify a per-machine installation, which implies a shared environment subject to change such as RDS/Pooled WVD and not a persistent, dedicated one.
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:156]:
```suggestion
    > These examples also use the **ALLUSERS=1** parameter. When you set this parameter, Teams Machine-Wide Installer appears in Programs and Features in Control Panel and in Apps & features in Windows Settings for all users of the computer. All users can then uninstall Teams if they have admin credentials. It's important to understand the difference between **ALLUSERS=1** and **ALLUSER=1**. The **ALLUSERS=1** parameter can be used in non-VDI and VDI environments and the **ALLUSER=1** parameter is used only in VDI environments to specify a per-machine installation, which implies a shared environment subject to change such as RDS/Pooled WVD and not a persistent, dedicated environment.
```

Comment (JohanFreelancer9) [reply Teams/teams-for-vdi.md:156]: Suggest changing the last word "one" to "environment".

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Per the PMs, the ALLUSER=1 parameter (setting per-machine installation mode for VDI) is not just for a shared environment. It can be used for both shared and dedicated VDI scenarios. The existing text is accurate as is and no changes are needed. Closing without merging.

**Pull request closed without merging**